README.md

```diff
@@ -1,37 +1,80 @@
 # FactorioWorld
 
-A mod for factorio that changes the map into a real-world map.
+A mod for factorio that changes the map into a real-world map.  You can choose between 'Atlantic' and 'Pacific' centered.  
 
-Note that it only changes the terrain, and not the spawning of resources. This is on my list of things to do, so maybe in the future!
+Note that it only changes the terrain, and not the spawning of resources. This is on the list of things to do, so maybe in the future!
 
 # Download
 
 You can get this mod here: https://mods.factorio.com/mods/TheOddler/factorio-world
 
 # Usage
 
-* Simply start a new game
-* Your world settings will be ignored, and the terrain replaced by the earth!
+* Start a new game
+* Choose a map layout - Atlantic or Pacific
+* Choose a spawn city
+* Map and terrain are generated from a map of Earth!
 
 # Settings
 
-Use the ingame mod settings to set:
+Use the in-game mod settings to set:
 
-* Map scale: Can be anything between 0.01 and infinite
+* World Map: Atlantic or Pacific. The difference is which is in the center of the world.  Detailed uses about 4x memory.
+* Map scaling factor: Can be anything between 0.01 and infinite (factors of 6 work well)
 * Spawn:
     * Pick a location to start, or set to custom
     * When set to custom, it'll use the custom spawn x and y coordinates
 * Repeat map: If set to false everything after the first iteration will become water
-* Use large map: If set to true the mod will use a more detailed map
-    * This will use about 4 times the memory I think
 * Safe zone size: All water within this zone will be turned into dirt
 
+# Available Spawn locations
+
+This is a list of the regions and cities | found in 'data/spAawns.lua':
+
+| Continent       | Region         | City        |
+|:----------------|:---------------|:------------|
+| Africa          | Mali           | Bamako      |
+| Africa          | Congo          | Katanga     |
+| Africa          | Chad           | Mongo       |
+| Africa          | Egypt          | Cairo       |
+| Africa          | Morocco        | Marrakech   |
+| Asia            | China          | Beijing     |
+| Asia            | China          | Kunming     |
+| Asia            | Georgia        | Tbilisi     |
+| Asia            | India          | Delhi       |
+| Asia            | Saudi Arabia   | Riyadh      |
+| Asia            | Mongolia       | Moron       |
+| Europe          | Czech Republic | Prague      |
+| Europe          | Russia         | Aktobe      |
+| Europe          | Russia         | Bilibino    |
+| Europe          | Russia         | Moscow      |
+| Europe          | Russia         | Omsk        |
+| Europe          | Russia         | Pechora     |
+| Europe          | Russia         | Yakutsk     |
+| Europe          | Spain          | Madrid      |
```

Comment:
I could remove this list from the ReadMe, but it is visible to people who haven't downloaded the mod, so it is a bit of advertising.  I'll update this to add London, but it could be removed if you think it adds not enough value to justify two copies.